stdlib/decimal.pyi

```diff
@@ -189,7 +189,6 @@ class Context:
         clamp: int | None = ...,
         flags: None | dict[_TrapType, bool] | Container[_TrapType] = ...,
         traps: None | dict[_TrapType, bool] | Container[_TrapType] = ...,
-        _ignored_flags: list[_TrapType] | None = ...,
```

Comment:
This one happened because this is an argument for `_pydecimal.Context`, but we should follow the C implementation here.